addon/mixins/model-records/relationships.js

```diff
@@ -19,13 +19,15 @@ function relationshipMacro(type) {
           records = Ember.A([records]);
         }
 
+        var store = _this.container.lookup('store:admin');
         var constructor = _this.get('model.constructor');
-        var inverse = constructor.inverseFor(property);
+        var inverse = constructor.inverseFor(property, store);
```

Comment:
Avoid `var` if you can. Prefer `const` for value references not expected to be mutated, else use `let`